src/core/directives/google-map-marker.ts

```diff
@@ -123,15 +131,18 @@ export class SebmGoogleMapMarker implements OnDestroy,
   }
 
   private _addEventListeners() {
-    this._markerManager.createEventObservable('click', this).subscribe(() => {
-      if (this._infoWindow != null) {
-        this._infoWindow.open();
-      }
-      this.markerClick.next(null);
-    });
+    if (this.openInfoWindow) {
+      this._markerManager.createEventObservable('click', this).subscribe(() => {
+        if (this._infoWindow != null) {
+          this._infoWindow.open();
+        }
+        this.markerClick.emit(null);
```

Comment:
This would introduce a bug. Now, the markerClick EventEmitter gets only emitted when openInfoWindow is true. This must get called outside the if statement.